pineappl/src/subgrid.rs

```diff
@@ -32,7 +33,7 @@ pub enum SubgridEnum {
 }
 
 /// Structure denoting renormalization and factorization scale values.
-#[derive(Debug, Deserialize, Clone, PartialEq, PartialOrd, Serialize)]
+#[derive(Debug, Deserialize, Clone, Copy, PartialEq, PartialOrd, Serialize)]
```

Comment:
Why/where do you need `Copy`?

Reply:
If I remove it I get the following error:
```
error[E0507]: cannot move out of a shared referenceineappl_py                                                                                                         
   --> pineappl_py/src/subgrid.rs:219:19
    |
219 |             .map(|&mu2| PyMu2 { mu2 })
    |                   ^---
    |                   ||
    |                   |data moved here
    |                   |move occurs because `mu2` has type `Mu2`, which does not implement the `Copy` trait
```

Reply:
In that case use say `.cloned()` before the `.map` call. You probably have to change `|&mu2|` to `|mu2|` (or `move |mu2|` ?).

Reply:
Ok thanks! [9fb4e10](https://github.com/NNPDF/pineappl/pull/222/commits/9fb4e105e7f15d11b9016ed81555bd3218a2ea79)